### PR TITLE
remove link to file

### PR DESCRIPTION
**Pull request by ArtificialOwl**
Replace 'file' type to 'highlight' to remove link to infected file.
- If the file was deleted, the link was already broken
- If the file was not deleted, the link would have download the file.

### CODE REVIEW

**Comment by ArtificialOwl** [comment]
image

**Review by skjnldsv — APPROVED** [review]

File: lib/Activity/Provider.php [hunk 1]
```diff
@@ -100,14 +100,16 @@ public function parse($language, IEvent $event, IEvent $previousEvent = null) {
 				'id' => $params[0],
 				'name' => $params[0],
 			];
+			$parameters['file'] = [
+				'type' => 'highlight',
+				'id' => $event->getObjectName(),
+				'name' => $event->getObjectName(),
```
Comment (come-nc) [inline lib/Activity/Provider.php:106]: You also changed from name to full path by not calling basename, not sure if intended (but given it’s not a link, full path is useful)

**Review by come-nc — APPROVED** [review]

**Comment by ArtificialOwl** [comment]
@come-nc @skjnldsv can I also have a review on https://github.com/nextcloud/files_antivirus/pull/307 that should fix the `Lint php-cs` error we are facing here !

**Comment by sol8712** [comment]
Thanks for the fix! You all are awesome!

**Pull request merged**